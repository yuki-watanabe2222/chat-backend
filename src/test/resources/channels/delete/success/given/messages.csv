id,channel_id,text,username,timestamp
202210151200-8097c0d2-ddc7-f02a-9dbf-29dfcd646d2b,1,今日のランチは焼き鳥,ユーザー,2022-10-15 12:00:00
202210151201-9f207c7a-16f5-80a0-a81f-5b216b9da38f,1,明日のランチは唐揚げ,ユーザー,2022-10-15 12:01:00
202210151202-469ffa6c-fcbf-7d95-cc8a-e1a4ba9a5b66,1,明日のディナーはよだれ鶏,ユーザー,2022-10-15 12:02:00
202210151203-1c575556-fec8-3ee8-64ba-67c2a351c6b3,2,チャンネル2のメッセージ,ユーザー,2022-10-15 12:03:00