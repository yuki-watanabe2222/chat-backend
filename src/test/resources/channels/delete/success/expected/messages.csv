id,channel_id,text,username,timestamp
202210151203-1c575556-fec8-3ee8-64ba-67c2a351c6b3,2,チャンネル2のメッセージ,ユーザー,2022-10-15 12:03:00